### PR TITLE
docs: upadeTamil

### PR DESCRIPTION
**Pull request by sabesansathananthan**

### CODE REVIEW

**Comment by sabesansathananthan** [comment]
@esau-morais Could you please review it.

File: translations/Translations.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+| <img alt="Ukrainian" title="Ukrainian" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/ua.svg" width="22">                   | [Українська](./README.ua.md)          |
+| <img alt="Russian" title="Russian" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/ru.svg" width="22">                       | [Русский](./README.ru.md)             |
+| <img alt="Italiano" title="Italiano" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/it.svg" width="22">                     | [Italiano](./README.it.md)            |
+| <img alt="Indian-Telugu" title="India-Telugu" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/in.svg" width="22">            | [Telugu](./README.te.md)              |
+| <img alt="Čeština" title="Čeština" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/cz.svg" width="22">                       | [Čeština](README.cs.md)               |
+| <img alt="SriLankan-Tamil" title="SriLankan-Tamil" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/lk.svg" width="22">       | [SL-Tamil](README.ta.md)              |
```
Comment (esau-morais) [inline translations/Translations.md:15]: Just to make sure is all good, change the route to `./README.ta.md` (optional)

**Review by esau-morais — COMMENTED** [review]
Thank you for update Tamil language!!

_Note: It look all good and following the [template](https://github.com/sabesansathananthan/React-GitHub-Resume/issues/15#issuecomment-704188333)_

File: translations/Translations.md (hunk 1 above)
Comment (sabesansathananthan) [reply translations/Translations.md]: I fixed this. any suggestions??

Comment (esau-morais) [reply translations/Translations.md]: All good, you can merge

Comment (sabesansathananthan) [reply translations/Translations.md]: Thank you for your Review. <3

**Pull request merged**